.github/workflows/trigger-anaconda.yml

```diff
@@ -44,8 +44,8 @@ jobs:
           for _ in $(seq 60); do
               sleep 60;
               if dnf copr enable -y packit/$COPR_NAME &&
-                 out=$(dnf info --refresh --repo='copr:*cockpit*' cockpit-bridge) &&
-                 stamp=$(echo "$out" | awk '/^Release/ { split($3, v, "."); print substr(v[2], 0, 14)}') &&
+                 out=$(dnf repoquery --refresh --repo='copr:*cockpit*' --queryformat '%{release}\n' --recent cockpit-bridge) &&
```

Comment:
In my F41 toolbox:
```
sudo dnf repoquery --refresh --queryformat '%{release}\n' --recent cockpit-bridge
```
gives empty output (just stderr from the refresh).  Without `--recent` I get `1.fc41` (which is useless for a released package, but I figure it's the right thing for the main COPR build). But I see your command output, so I trust it actually works.